packages/core/src/state-management/stream-from-state.ts

```diff
@@ -1,19 +1,19 @@
-import { Context, StreamState, Stream } from '@ceramicnetwork/common'
+import { StreamState, Stream, StreamReaderWriter } from '@ceramicnetwork/common'
 import { Observable } from 'rxjs'
 import { HandlersMap } from '../handlers-map.js'
 import { StateLink } from './state-link.js'
 
 /**
  * Build Stream from the current state and update feed.
  *
- * @param context - Ceramic context
+ * @param context - Interface for reading and writing to ceramic network
  * @param handlersMap - available stream handlers
  * @param state - current state of the stream
  * @param update$ - On-demand feed of updates for the stream. If not provided then the returned
  *   Stream object is marked read-only and cannot be used to update the stream.
  */
 export function streamFromState<T extends Stream>(
-  context: Context,
+  context: StreamReaderWriter,
```

Comment:
should we rename this arg now?  Same for the other places we take a StreamReader or StreamReaderWriter and call it context

Reply:
Yes, but not in this PR. Minimizing renaming in this PR to reduce scope.